app/src/androidTest/java/com/twilio/video/app/util/UiAutomatorExtensions.kt

```diff
@@ -0,0 +1,27 @@
+package com.twilio.video.app.util
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+import timber.log.Timber
+
+fun uiDevice(): UiDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+
+fun UiDevice.clearTask(swipeAttempts: Int = 10 ) {
```

Comment:
So cool 😂

Reply:
Thanks! I was really happy I got this working. Really fun to watch the test run and attempt the swipes 🙂